pkg/crd/markers/crd.go

```diff
@@ -419,3 +426,183 @@ func (s SelectableField) ApplyToCRD(crd *apiext.CustomResourceDefinitionSpec, ve
 
 	return nil
 }
+
+// +controllertools:marker:generateHelp:category=CRD
+
+// SchemaModifier allows modifying JSONSchemaProps for CRDs.
+//
+// The PathPattern field defines the rule for selecting target fields within the CRD structure.
+// This rule is specified as a path in a JSONPath-like format and supports special wildcard characters:
+// - `*`: matches any single field name (e.g., `/spec/*/field`).
+// - `**`: matches fields at any depth, across multiple levels of nesting (e.g., `/spec/**/field`).
+//
+// Example:
+// +kubebuilder:schemaModifier:pathPattern=/spec/exampleField/*,description=""
+//
+// In this example, all fields matching the path `/spec/exampleField/*` will have the empty description applied.
+//
+// Any specified values (e.g., Description, Format, Maximum, etc.) will be applied to all schemas matching the given path.
+type SchemaModifier struct {
```

Comment:
Is the list of fields comprehensive, and, if so, how can we ensure in the future, if additional fields are needed, that we do not forget to add them?